README.md

```diff
@@ -1,70 +1,26 @@
-# Quick Start
```

Comment:
Are you accidentally undo-ing @Ninjagod1251 's changes here?
Are you not caught up with main?

Reply:
Yeah, has to rebase on main

Reply:
Yeah, I'm not sure what happened when I went to rebase, but it didn't show me the differences like it should have and showed them in reverse for some of them

Reply:
> Yeah, I'm not sure what happened when I went to rebase, but it didn't show me the differences like it should have and showed them in reverse for some of them

Some of your commits are probably older then.

first make sure your version `main` is up to date with `upstream`, then checkout this branch again and do `git rebase -i main` and remove any commits that you didn't mean to add. finally, do `git push -f` if you are happy with the update only containing changes you've made based off of `main`